src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -122,6 +122,23 @@ public FeedbackSessionResultsBundle(FeedbackSessionAttributes feedbackSession,
         // roster.*Table is populated using the CourseRoster data directly
         this.rosterTeamNameMembersTable = getTeamNameToEmailsTableFromRoster(roster);
         this.rosterSectionTeamNameTable = getSectionToTeamNamesFromRoster(roster);
+        if (!isAllQuestionNumbersConsistent()) {
+            log.severe(feedbackSession.getIdentificationString() + " has invalid question numbers");
+        }
+    }
+    
+
+    //TODO can be removed once we are sure that question numbers will be consistent
+    private boolean isAllQuestionNumbersConsistent() {
+        Set<Integer> questionNumbersInSession = new HashSet<>();
+        for (FeedbackQuestionAttributes question : this.questions.values()) {
```

Comment:
Anyway dude, this checks whether the same question number appears twice. Is that the only way? can't there be gaps in question numbers? would that be a problem? I am not sure, just throughing some questions haha